src/FacadeAccountCreation/FacadeAccountCreation.Core/Models/Organisations/PaymentCalculationResponse.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace FacadeAccountCreation.Core.Models.Organisations
+{
+    [ExcludeFromCodeCoverage]
+    public class PaymentCalculationResponse
+    {
+        public decimal ProducerRegistrationFee { get; set; }
```

Comment:
No need for return type to be decimal because all amounts will be in _pence_. So the return value should be **int**.